src/controller/postRouter.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const { createPost } = require('../services/postService');
+const { authenticateToken } = require('../middleware/authMiddleware');
+const { validateTextBody, validateScore } = require('../middleware/postMiddleware');
+
+
+const postRouter = express.Router();
+
+postRouter.post("/create", authenticateToken, validateTextBody, validateScore, async (req, res) => {
+    //TODO check song title exists in API
+    try {
+        await createPost(req.user.Username, req.body.Text, req.body.Score, "TTT");
```

Comment:
`user` here is lowercase, even though in `authMiddleware` it is capitalized; I'll change everything to camel case next sprint to make everything more consistent, so just make sure they have matching capitalization for now.